cpu/atmega256rfr2/include/cpu_conf.h

```diff
@@ -1,51 +0,0 @@
-/*
- * Copyright (C) 2017 RWTH Aachen, Josua Arndt
- *
- * This file is subject to the terms and conditions of the GNU Lesser
- * General Public License v2.1. See the file LICENSE in the top level
- * directory for more details.
- */
-
-/**
- * @ingroup         cpu_atmega256rfr2
- * @{
- *
- * @file
- * @brief           Implementation specific CPU configuration options
- *
- * @author          Josua Arndt <jarndt@ias.rwth-aachen.de>
- * @author          Steffen Robertz <steffen.robertz@rwth-aachen.de>
- */
-
-#ifndef CPU_CONF_H
-#define CPU_CONF_H
-
-#include "atmega_regs_common.h"
-
-#ifdef __cplusplus
-extern "C" {
-#endif
-
-/**
- * @name Kernel configuration
- *
- * Since printf seems to get memory allocated by the linker/avr-libc the stack
- * size tested sucessfully even with pretty small stacks.k
- * @{
- */
-
-/* keep THREAD_STACKSIZE_IDLE > THREAD_EXTRA_STACKSIZE_PRINTF
- * to avoid not printing of debug in interrupts
- */
-#define THREAD_EXTRA_STACKSIZE_PRINTF    (128)
-
-#ifndef THREAD_STACKSIZE_DEFAULT
-#define THREAD_STACKSIZE_DEFAULT   (512)
-#endif
-
-#define THREAD_STACKSIZE_IDLE      (129)
```

Comment:
Note change of behavior: typo fixed.

Reply:
It wasn't a typo actually. The authors of this port (@Josar and @shr70) claim that with this change `printf()` works from the interrupt context. Can you guys confirm?

Reply:
If i remember correctly, I think it has to do with this line.

https://github.com/RIOT-OS/RIOT/blob/af829e2da41982bc903fe5e0149745cfea7f863c/core/include/debug.h#L55

If you use `DEVELHELP` and there are interrupts when in idle thread there will just be the message
```
"Cannot debug, stack too small")
```

Reply:
I agree with @ZetaR60, and should be done in a folo. I can take care of it when this PR is merged.